pkg/manager/manager.go

```diff
@@ -73,7 +70,6 @@ func New(opts Options) (Manager, error) {
 		SyncPeriod:              &opts.SyncPeriod,
 		Namespace:               opts.WatchNamespace,
 		NewCache:                opts.NewCache,
-		NewClient:               newClientFunc,
```

Comment:
This is not needed anymore, it is a default option with the newer version of `controller-runtime`